src/utils/buildEdlGroupPath.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Returns a EDL api path based on the provided type, id and params.
+ * @param {Object} params
+ * @param {String} params.id ID parameter to send to EDL
+ * @param {String} params.params Query parameters to send to EDL
+ * @param {String} params.type Request type to send to EDL
+ */
+export const buildEdlGroupPath = ({
+  id,
+  params,
+  type
+}) => {
+  const typeMap = {
```

Comment:
I mentioned using an array for the constant, but maybe we just need to combine that constant with this util? The constant seems redundant in that it defines keys and values that are the same. I think

Reply:
Discussed this on a call, not an issue